compiler/src/dmd/dcast.d

```diff
@@ -704,7 +704,7 @@ MATCH implicitConvTo(Expression e, Type t)
                     return MATCH.nomatch;
                 m = MATCH.constant;
             }
-            if (e.hexString && tn.isintegral && (tn.size == e.sz || (!e.committed && (e.len % tn.size) == 0)))
+            if (e.hexString && tn.isintegral && (tn.size == e.sz || (!e.committed && e.len % tn.size == 0)))
```

Comment:
I like all of these, except this. The parenthesis make it obvious what is happening.